crates/key-value-redis/Cargo.toml

```diff
@@ -12,6 +12,7 @@ spin-core = { path = "../core" }
 spin-factor-key-value = { path = "../factor-key-value" }
 tokio = { workspace = true }
 url = { workspace = true }
+log = "0.4.22"
```

Comment:
I think we use `tracing` rather than `log` in the runtime.  `log` is sometimes used in tests.